lib/internal/Magento/Framework/Setup/Patch/PatchApplier.php

```diff
@@ -292,4 +290,20 @@ public function revertDataPatches($moduleName = null)
             }
         }
     }
+
+    /**
+     * Checks is patch was applied with current alias name
+     *
+     * @param DataPatchInterface|SchemaPatchInterface $dataPatch
+     * @return bool
+     */
+    private function checkPatchAliases($dataPatch): bool
+    {
+        if ($dataPatchAliases = $dataPatch->getAliases()) {
+            foreach ($dataPatchAliases as $patchAlias) {
+                return $this->patchHistory->isApplied($patchAlias) ?? true;
```

Comment:
@Usik2203 I don't see how this will work if you have more than one alias. This line will cause the function to return on the first test, no matter what, when it should only return if it finds an alias that is applied (and then default to false if none have been installed).
cc: @rogyar